README.md

```diff
@@ -8,21 +8,36 @@ If you have questions, or wish to contribute to iTwin.js, see our [Contributing
 
 ## About this Repository
 
-Contains the __@itwin/insights-client__ package that wraps sending requests to the project service. Visit the [Insights API](https://developer.bentley.com/apis/insights/) and [Carbon calculation API](https://developer.bentley.com/apis/carbon-calculation/) for more documentation on the insights service.
+Contains the __@itwin/insights-client__ package that wraps sending requests to the reporting service. Visit the [Insights API](https://developer.bentley.com/apis/insights/) and [Carbon calculation API](https://developer.bentley.com/apis/carbon-calculation/) for more documentation on the insights service.
 
 ## Environment Variables
 
-```
-# ---- Optional URL prefix for dev/qa environments ----
+### Optional URL prefix for dev/qa environments
 IMJS_URL_PREFIX=""
```

Comment:
Surround IMJS_URL_PREFIX in triple ` quotes

Reply:
surrounded (in the line above and below)